.github/workflows/staging-orchestrator.yml

```
@@ -124,3 +108,34 @@ jobs:
+
+      - name: Generate all dependency diagrams
+        run: pnpm run diagrams:all
+
+      - name: Commit diagram outputs
+        uses: stefanzweifel/git-auto-commit-action@v6
+        with:
+          file_pattern: diagrams/output/current/*
+          commit_message: "chore(diagrams): refresh dependency outputs [skip ci]"
+          commit_user_name: github-actions[bot]
+          commit_user_email: github-actions[bot]@users.noreply.github.com
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)